test/test_force.js

```diff
@@ -129,28 +129,30 @@ function usage(exitCode) {
     utils.logInfo('  - appTypeX is: native, native_swift, react_native, hybrid_local or hybrid_remote', COLOR.cyan);
     utils.logInfo('  - templaterepouri is a template repo uri e.g. https://github.com/forcedotcom/SmartSyncExplorerReactNative', COLOR.cyan);
     utils.logInfo('', COLOR.cyan);
-    utils.logInfo('  If hybrid is targeted, the following are first done:', COLOR.cyan);
+    utils.logInfo('  If hybrid is targeted:', COLOR.cyan);
```

Comment:
NB: test_force.js takes the same arguments as before, only the explaining message has changed a bit.